app.py

```diff
@@ -39,75 +52,109 @@ def daily():
 	except ValueError:
 		# Default to today (at midnight) if not
 		date = datetime.combine(datetime.today(), datetime.min.time())
-		
+	
 	# Formatted date strings when rendering page and buttons to other dates
 	date_str = date.strftime('%Y-%m-%d')
 	prev_date_str = (date-timedelta(1)).strftime('%Y-%m-%d')
 	next_date_str = (date+timedelta(1)).strftime('%Y-%m-%d')
 	
-	print("==== " + date_str + " ====" + "="*100)
+	tab_list = client_format.keys()
 	
-	# Determines what each tab/graph should display
-	with open(CLIENT_FORMAT_FILE) as file_handle:
-		client_format = json.load(file_handle)
-	
-	# Specifies information about each sensor in the database
-	with open(DATABASE_FORMAT_FILE) as file_handle:
-		db_format = json.load(file_handle)
+	try:
+		tab = request.args.get('tab')
+		if not tab in client_format.keys(): raise ValueError()
+	except ValueError:
+		tab = next(iter(client_format))
 	
+	#print("==== " + date_str: tab + " ====" + "="*100)	
```

Comment:
This comment might have been left in by accident - if not then that's totally cool.